### DISCUSSION TITLE
版本管理--代码对比

### DISCUSSION THREAD

**Discussion by hardstuding** [apache/dolphinscheduler#16508] [Ideas]
在版本管理的时候，虽然会做各个版本的备注，但是我还是需要对比各个版本代码间的细节，当前工具好像只支持直接切换版本，无法在UI上快速的对比不同版本间的代码差异。
是我没有发现功能怎么使用吗？或者开发团队有计划开发此功能吗？

**Comment by fangshirui** [comment]
诶诶，我也遇到这个问题了
